### PR TITLE
update required dependency

### PR DESCRIPTION
**Pull request by kuri-leo**
Fix https://github.com/tensorflow/tensorflow/issues/36588

[TF 2.1 release](https://github.com/tensorflow/tensorflow/releases/tag/v2.1.0) brings in this new dependency requirement

### CODE REVIEW

File: site/en/install/pip.html [hunk 1]
```diff
@@ -32,7 +32,8 @@ <h3>System requirements</h3>
   <li>macOS 10.12.6 (Sierra) or later (64-bit) <em>(no GPU support)</em></li>
   <li>Windows 7 or later (64-bit) <em>(Python 3 only)</em></li>
```
Comment (lamberta) [inline site/en/install/pip.html]: Perhaps add as a bullet point under Windows.
We already do note this in the Windows tab selector.

Comment (kuri-leo) [reply site/en/install/pip.html]: Just fixed that, is this right? (plz forgive my poor Html code 🤦‍♂

Comment (lamberta) [reply site/en/install/pip.html]: Almost! Webmaster-in-training 😄

Comment (kuri-leo) [reply site/en/install/pip.html]: Wow, thanks a lot! 👏

**Pull request merged**